app/controllers/automated_tests_controller.rb

```diff
@@ -91,8 +91,17 @@ def populate_autotest_manager
         { key: "#{file}/" }
       else
         file_keys << file
-        { key: file, size: 1,
-          url: download_file_assignment_automated_tests_url(assignment_id: assignment.id, file_name: file) }
+
+        if files_dir.join(file).exist?
+          date = File.mtime(files_dir.join(file))
+          local_date = date.in_time_zone(current_user.time_zone)
```

Comment:
Hmmm, I wonder if calling `in_time_zone` is still necessary after using `I18n.l` (see comment below). Could you look into this? If you used similar code from elsewhere, it might be that we can simplify that too, in a separate PR.